client/src/app/site/pages/meetings/services/active-meeting.service.ts

```diff
@@ -35,7 +35,7 @@ const getMeetingDetailSubscriptionConfig = (id: Id, getNextMeetingIdObservable:
         follow: [
             `chat_group_ids`,
             `chat_message_ids`,
-            { idField: `poll_ids` },
+            { idField: `poll_ids`, follow: [`content_object_id`] },
```

Comment:
Why is this necessary?

Reply:
Because I need the contentobject to be loaded to correctly get the data for the banner